crm-plugin/controllers/crm-plugin.js

```diff
@@ -1,11 +1,14 @@
 "use strict";
-
 /**
  * crm-plugin.js controller
  *
  * @description: A set of functions called "actions" of the `crm-plugin` plugin.
  */
 const { sanitizeEntity } = require("strapi-utils");
+const _utils = require("../services/utils.js");
```

Comment:
do not use import. use proper services method to get data refer below link
https://strapi.io/documentation/3.0.0-beta.x/concepts/services.html#custom-services